doc/user/content/sql/functions/to_char.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "to_char Function"
+description: "Converts a timestamp into a string using the specified format."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`to_char` returns a string created using the specified format string.
+
+The format string can be composed of any number of format tokens, interspersed with
+regular text. You can place a format token inside of double-quotes to emit it literally.
+
+## Examples
+
+**RFC 2822 Format**
+
+```sql
+SELECT to_char(TIMESTAMPTZ '2019-11-26 15:56:46 +00:00', 'Dy, Mon DD YYYY HH24:MI:SS +0000') AS formatted
+```
+```nofmt
+             formatted
+ ---------------------------------
+  Tue, Nov 26 2019 15:56:46 +0000
+```
+
+**Additional non-interpreted Text**
+
+```sql
+SELECT to_char(TIMESTAMPTZ '2019-11-26 15:56:46 +00:00', '"Welcome" to Mon, YYYY') AS formatted
+```
+```nofmt
+       formatted
+ ----------------------
+  Welcome to Nov, 2019
+```
+
+**Ordinal Modifiers**
+
+```sql
+SELECT to_char(TIMESTAMPTZ '2019-11-1 15:56:46 +00:00', 'Dth of Mon') AS formatted
+```
+```nofmt
+  formatted
+ ------------
+  6th of Nov
+```
+
+## Format Specifiers
+
+Regular specifiers:
+
+| Pattern                     | Description                                                                                      |
+|-----------------------------|--------------------------------------------------------------------------------------------------|
+| `HH`                        | hour of day (01-12)                                                                              |
+| `HH12`                      | hour of day (01-12)                                                                              |
+| `HH24`                      | hour of day (00-23)                                                                              |
+| `MI`                        | minute (00-59)                                                                                   |
+| `SS`                        | second (00-59)                                                                                   |
+| `MS`                        | millisecond (000-999)                                                                            |
+| `US`                        | microsecond (000000-999999)                                                                      |
+| `SSSS`                      | seconds past midnight (0-86399)                                                                  |
+| `AM`/`am`/`PM`/`pm`         | AM, am, PM, or pm meridiem indicator (without periods)                                           |
```

Comment:
The description for this leaves me with some questions about what this does, e.g. are the meridiem indicators here case-sensitive? If not maybe stress that the pattern is case insensitive and does not accept periods.

Reply:
The Postgres docs just say "meridiem indicator (without periods)", which I actually think is more clear despite being more terse. The phrasing here kind of makes it seem like if you write `pm` you'll only ever get an "pm" as output. Maybe we should split it out into all four cases?

```
`AM` or `PM` | uppercase meridiem indicator without periods
`am` or `pm` | lowercase meridiem indicator without periods
`A.M.` or `P.M.` | uppercase meridiem indicator with periods
`a.m.` or `p.m.` | lowercase meridiem indicator with periods 
````